lib/usd/translators/jointWriter.cpp

```diff
@@ -197,19 +198,20 @@ static bool _FindDagPoseMembers(
             MObjectHandle connNode(inputs[j].node());
             auto          it = pathIndexMap.find(connNode);
             if (it != pathIndexMap.end()) {
-                (*indices)[it->second] = i;
+                indices[it->second] = i;
             }
         }
     }
 
     // Validate that all of the input dagPaths are members.
-    for (size_t i = 0; i < indices->size(); ++i) {
-        int index = (*indices)[i];
+    for (size_t i = 0; i < dagPaths.size(); ++i) {
```

Comment:
If I'm on the right track with the comments above, then I think it's still appropriate to iterate over `indices` here rather than `dagPaths`, since they will both be the same size, and `indices` is the thing we've been filling with data.

Reply:
This is also now reverted since we are betting on the assumption that `dagPaths`
and `indices` must be the same size and that the `pathIndexMap`'s 2nd member
will not return a value larger than said size.